google/cloud/bigtable/data/_async/client.py

```diff
@@ -442,6 +443,16 @@ def __init__(
         default_mutate_rows_attempt_timeout: float | None = 60,
         default_operation_timeout: float = 60,
         default_attempt_timeout: float | None = 20,
+        default_read_rows_retryable_errors: Sequence[
+            grpc.StatusCode | int | type[Exception]
+        ] = (DeadlineExceeded, ServiceUnavailable, Aborted),
+        default_mutate_rows_retryable_errors: Sequence[
+            grpc.StatusCode | int | type[Exception]
+        ] = (DeadlineExceeded, ServiceUnavailable),
+        default_retryable_errors: Sequence[grpc.StatusCode | int | type[Exception]] = (
```

Comment:
I would suggest to pick one way to express error types and stick with it. I dont think there much to gain from this flexibility

Reply:
Are you ok with sticking with `type[Exception]`? That's what `api-core` uses in retries, so it would be simplest. But I thought you mentioned that it should accept StatusCodes at one point